resources/user-agents/browsers/safari/safari-4-0.json

```diff
@@ -27,10 +27,10 @@
       },
       "children": [
         {
-          "match": "Mozilla/5.0 (SMART-TV; X11;#PLATFORM#) applewebkit* (*khtml*like*gecko*) *Version/#MAJORVER#.#MINORVER#* Safari/*",
-          "device": "Smart-TV",
+          "match": "Mozilla/5.0 (#PLATFORM#)*applewebkit*(*khtml*like*gecko*)*Version/#MAJORVER#.#MINORVER#*Safari/*",
```

Comment:
I did add Safari 4.1 on linux here as it seems possible to install Safari on linux using Wine, the caveat is that it has to be the Windows version of Safari, which only had a few releases (3.0 - 5.1): https://en.wikipedia.org/wiki/Safari_version_history#Windows